Components/Navigation/navigation.css

```diff
@@ -0,0 +1,26 @@
+.nav-bar{
+    background-color: tomato;
```

Comment:
Try to use variables for colors, they help if you need to change the color later on